crates/lang/macro/src/lib.rs

```diff
@@ -136,8 +136,14 @@ pub fn selector_bytes(input: TokenStream) -> TokenStream {
 ///
 ///     An example for this is the following type that could potentially be used
 ///     within a contract's storage struct definition:
-///     ```
-///     // A storage vector of storage vectors.
+///
+///
+///     ```ignore
+///     # // TODO [#XYZ]: Right now we've hidden the `StorageVec` from public access so this
```

Comment:
Is this PR work in progress? Otherwise please no TODOs in code. Should create an issue for this instead.

Reply:
That's what the `#XYZ` is supposed to indicate, the number of the issue. I didn't want to jump the gun on creating the issues :)